src/debugAdapter/goDebug.ts

```diff
@@ -181,7 +181,7 @@ class Delve {
 		this.program = program;
 		this.remotePath = remotePath;
 		let mode = launchArgs.mode;
-		let env = launchArgs.env || {};
+		let env = Object.assign({}, launchArgs.env, process.env);
```

Comment:
this way the `process.env` overrides `launchArgs.env` 
It should be the other way around